models/issues/pull.go

```diff
@@ -448,6 +448,24 @@ func (pr *PullRequest) GetGitHeadBranchRefName() string {
 	return fmt.Sprintf("%s%s", git.BranchPrefix, pr.HeadBranch)
 }
 
+func (pr *PullRequest) GetReviewCommentsCount(ctx context.Context) int {
+	opts := FindCommentsOptions{
```

Comment:
other approach needed.

what exactly are ReviewComments?
-> code comments
-> reviews itselve
-> ...

Reply:
After some research I found the following:
[review_comments](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28) is the number of all review_comments on a pull request.
I verified that by also calling the github rest-api on a test repo with 2 review comments.
```
curl -L \
  -X PATCH \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <token>" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/Chief-Detektor/test/pulls/6 \
  -d '{"title":"new title","body":"updated body","state":"open","base":"main"}'
```
returns:
```
...
 "comments": 0,
  "review_comments": 2,
  "maintainer_can_modify": false,
  "commits": 1,
  "additions": 1,
  "deletions": 5,
  "changed_files": 1
}
```
Also these numbers are submitted via the pull_request webhook.

Reply:
I tested this again on gitea with my changes and the behavior is identical.